Update index.md (#5515)
segmentio/segment-docs: src/connections/sources/catalog/cloud-apps/alloy-automation/index.md

---
title: 'Alloy Automation Source'
hidden: true

> ---

Reviewer: ```suggestion
id: tisa2A0jAs
beta: true
---
```

# AlloyFlow Source

> [Alloy](https://runalloy.com/flow/) is a specialized no-code automation platform, built with e-commerce businesses in mind. It seamlessly integrates various applications, paving the way for a streamlined shopping experience and efficient operational workflows.

Reviewer: ```suggestion
[Alloy](https://runalloy.com/flow/){:target="_blank"} is a specialized no-code automation platform, built with e-commerce businesses in mind. It seamlessly integrates various applications, paving the way for a streamlined shopping experience and efficient operational workflows.
```

Combining Alloy's automation prowess with Segment's data collection and distribution capabilities, businesses can capture and react to real-time customer activities across all digital interfaces. Whether it's a website visit, a mobile app interaction, or server-side processes, Segment simplifies data collection. With Alloy in the mix, these interactions can automatically trigger tailored workflows, connecting apps, and enhancing customer experiences.

Using Segment, the interaction data captured can be sent directly to marketing, product, and analytics tools. The integration with Alloy means that this data can also be utilized to drive automation, ensuring consistent and personalized user interactions across all touchpoints.

> 

Reviewer: ```suggestion

This source is maintained by Alloy. For any issues with the source, [Contact the Alloy Support team](mailto:<add support email>)
```

Please provide a mailing address or link to support page^

Author: contact@runalloy.com

## Getting Started

> 1. **Set Up the Segment Block**: Begin by setting up the Segment block within Alloy. This will enable Alloy to forward data to Segment efficiently. If you're unfamiliar with how to do this, the Alloy documentation or support team can guide you through the process.

Reviewer: Can you provide a link for "the Alloy documentation"?

Author: https://docs.runalloy.com/docs

> 1. **Set Up the Segment Block**: Begin by setting up the Segment block within Alloy. This will enable Alloy to forward data to Segment efficiently. If you're unfamiliar with how to do this, the Alloy documentation or support team can guide you through the process.
> 
> 2. **Contact the Alloy Team**: If you prefer a hands-off approach, you can request the Alloy team to set up the Segment integration on your behalf. They possess the expertise to ensure a smooth data flow from Alloy to Segment.

Reviewer: ```suggestion
1. From your workspace's [Sources catalog page](https://app.segment.com/goto-my-workspace/sources/catalog){:target="_blank”} click **Add Source**.
2. Search for "Ally Automation" in the Sources Catalog, select it, and click **Add Source**.
3. Give the source a name, for example Alloy_Prod. The name is used in the Segment interface. Click add source to save your settings.
4. Copy the Write key from the Segment UI.
5. In Alloy, **Set Up the Segment Block**: This will enable Alloy to forward data to Segment efficiently. If you're unfamiliar with how to do this, the Alloy documentation or support team can guide you through the process.
6. **Contact the Alloy Team**: If you prefer a hands-off approach, you can request the Alloy team to set up the Segment integration on your behalf. They possess the expertise to ensure a smooth data flow from Alloy to Segment.
```

Author: looks to be a typo in Alloy missing the o

> ## Types of Calls to Segment

Reviewer: Are the events you send to Segment configured by the user? In other words, are there no events that alloy sends to segment b default? 

If there are default events, you will need to create a table like so listing out the events that are sent. 

- https://github.com/segmentio/segment-docs/blob/develop/templates/partners/source.md#events

Author: There are no default events, the user must select which ones.

Alloy's integration with Segment primarily supports the following calls:

- **Track**: Captures the actions users are taking within your e-commerce platform and sends the information to Segment.
- **Identify**: Recognizes and forwards user identities, helping Segment collate data on individual users across sessions.

> ## Support
> 
> This source is maintained by Alloy. For any inquiries or challenges related to the source, the Alloy support team is readily available.

Reviewer: ```suggestion
```

Moved to top of page to be inline with our doc spec
